Parsers/ASimAuthentication/Parsers/ASimAuthenticationSentinelOne.yaml

```diff
@@ -0,0 +1,185 @@
+Parser:
```

Comment:
@jayeshprajapaticrest did you find any alert logs from SentinelOne any schema related?

Logs which can be mapped to these fields with https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-authentication#inspection-fields

We have these fields in each schema

Reply:
@jayeshprajapaticrest 
No other Threat field info coming in SentinelOne Authentication?
The logs I had only have 'threatconfidence' as '15', hope you have validated with various sample values i.e. 75, 80, 90, 100 etc.

Reply:
Author commented on other PR that they didn't had actual data but have tested it by ingesting curated test data